pkg/backup/compaction_test.go

```diff
@@ -675,6 +677,89 @@ func TestBackupCompactionUnsupportedOptions(t *testing.T) {
 	}
 }
 
+func TestCompactionCheckpointing(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	// Need a large enough backup so that it doesn't finish in one iteration
```

Comment:
wdym here?

Reply:
It's something I saw we used in our checkpoint tests, but basically I need to ensure that the backup
is large enough that checkpoints will be created in the first place.

Reply:
chatted offline. i'd like this comment to be a bit more clear.